lib/block-supports/behaviors.php

```diff
@@ -84,17 +84,22 @@ function gutenberg_render_behaviors_support_lightbox( $block_content, $block ) {
 
 	$aria_label = __( 'Enlarge image', 'gutenberg' );
 
+	$processor->next_tag( 'img' );
 	$alt_attribute = $processor->get_attribute( 'alt' );
 
+	// An empty alt attribute `alt=""` is valid for decorative images.
```

Comment:
excellent comment, and excellent fix above. one small note: `null` indicates that the `alt` attribute is missing, but an empty value would be `''` or if it's only present as a name like `src="…" alt loading=eager` then the value would be `true`

```php
if ( is_string( $alt_attribute ) ) {
```

this might be more appropriate, as it ensures that that we're dealing with an `alt=something` situation

Reply:
In a way, the check is a  bit redundant in the first place. As far as I know, WordPress enforces at least an empty `alt=""` attribute everywhere. I'd agree to check for string anyways, just in case.